packages/common/errors.js

```diff
@@ -9,105 +11,37 @@ const isFunction = require('lodash.isfunction');
  */
 
 const createErrorType = (name, ParentType = Error) => {
-  function E(message) {
-    if (isFunction(Error.captureStackTrace)) {
-      Error.captureStackTrace(this, this.constructor);
-    } else {
-      this.stack = (new Error(message)).stack;
-    }
-    this.message = message;
-  }
-  E.prototype = new ParentType();
-  E.prototype.name = name;
-  E.prototype.constructor = E;
-  return E;
+  deprecate('@cumulus/common/errors/createErrorType', '1.17.1', '@cumulus/errors/createErrorType');
+  return errors.createErrorType(name, ParentType);
 };
 
-const WorkflowError = createErrorType('WorkflowError');
-
 /**
  * Returns true if the error is a resource error.
  *
  * @param {Error} error
  * @returns {boolean}
  */
-const isWorkflowError = (error) => error.name.includes('WorkflowError');
+const isWorkflowError = (error) => {
+  deprecate('@cumulus/common/errors/isWorkflowError', '1.17.1', '@cumulus/errors/isWorkflowError');
+  return errors.isWorkflowError(error);
```

Comment:
If this is in `errors`, it would seem to make sense that `isThrottlingException` would also be in `errors`.